### PR TITLE
Git export to use git cli \o/

### PR DESCRIPTION
**Pull request by wundervaflja**

### CODE REVIEW

File: git_stats.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#! /bin/env python
+
+import sys
+data = sys.stdin.read()
+data = [k.split('\n\n') for k in [i.strip() for i in data.split('COMMITBEGIN')] if k]
```
Comment (Volodymyr-Boiko) [inline git_stats.py:7]: a little bit hard to read and understand what is going on here

File: git_stats.py [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    if len(i) == 1:
+        stats[i[0]] = {}
+    else:
+        inner_stats = []
+        for stat in i[1].split('\n'):
+            stat = stat.split('\t')
+            inner_stats.append({'insertions':stat[0], 'deletions':stat[1],'path':stat[2]})
+        stats[i[0]] = inner_stats
+
+print(stats)
```
Comment (Volodymyr-Boiko) [inline git_stats.py]: do we need this print?

Comment (wundervaflja) [reply git_stats.py]: yes - for stdout

File: git_stats.py (hunk 1 above)
Comment (wundervaflja) [reply git_stats.py:7]: will improve

**Pull request merged**